.github/workflows/ci.yml

```diff
@@ -176,6 +176,8 @@ jobs:
       - uses: ./.github/actions/upload
         with:
           test-report-retention-days: ${{ env.TEST_REPORT_RETENTION_DAYS }}
+          # only upload heap dumps if running on forks, where no secrets are used
+          upload-heap-dump: ${{ secrets.SECRETS_PRESENT != 'present' }}
```

Comment:
maybe `upload-heap-dump: ${{ github.repository_owner == 'trinodb' }}` is simpler. Does it work?

Reply:
We'd only want to upload heap dumps in forks, where there are no secrets set. I'd add this as an additional condition, but it'd be reversed.